spec/mysql2/statement_spec.rb

```diff
@@ -712,7 +712,6 @@ def stmt_count
 
     it 'should return number of rows affected by an insert' do
       stmt = @client.prepare 'INSERT INTO lastIdTest (blah) VALUES (?)'
-      expect(stmt.affected_rows).to eq 0
```

Comment:
As commented: The change in the spec is required for mariadbclient. It throws an error if no query was executed.